screens/MealDetailsScreen.js

```diff
@@ -0,0 +1,92 @@
+/******************************************
+ *  Author : Rotems
+ *  Created On : Tue Mar 31 2020
+ *******************************************/
+
+import React from "react";
+import { StyleSheet, View, Text, ScrollView, Image } from "react-native";
+import { MEALS } from "../data/dummy-data";
+import Icon from "react-native-vector-icons/MaterialIcons";
+import Color from "../constants/Colors";
+import LineText from "../components/LineText";
+import MealDetailCircle from "../components/MealDetailCircle";
+
+const MealDetailsScreen = props => {
+  const mealName = MEALS[0].title;
+  const complexity = MEALS[0].complexity;
+  const duration = MEALS[0].duration;
+  const isDairy = MEALS[0].filters.isDairy;
+  const isMeat = MEALS[0].filters.isMeat;
+
+  let kosher = "פרווה";
```

Comment:
This should only happen one time and not every time the component gets compiled.